app/api/groups/remove_user_batch.go

```diff
@@ -15,26 +15,34 @@ import (
 // swagger:operation DELETE /user-batches/{group_prefix}/{custom_prefix} groups userBatchRemove
 // ---
 // summary: Remove a user batch
-// description:
-//   Lets an admin remove user batches and all users having "{group_prefix}_{custom_prefix}_" as login prefix.
+// description: |
+//   Lets a group manager remove user batches and all users having "{group_prefix}_{custom_prefix}_" as login prefix.
 //
-//   On success the service
 //
-//     * requests the login module to delete the users with {group_prefix}_{custom_prefix}_" as prefix
-//       (/platform_api/accounts_manager/delete with prefix parameter);
+//   If the preconditions are satisfied, the service
 //
-//     * deletes all users with "{group_prefix}_{custom_prefix}_" as prefix
+//     * requests the login module to delete the users with "{group_prefix}\_{custom_prefix}\_" as prefix
+//       (/platform_api/accounts_manager/delete with the `prefix` parameter);
+//
+//     * deletes all users with "{group_prefix}\_{custom_prefix}\_" as prefix
 //       (ignoring the membership locks on groups the authenticated user manage (but not others!));
```

Comment:
manage**s** (missing "s")
(and maybe add a "that"?) -> "(ignoring the membership locks on groups that the authenticated user manages (but not others!));"

Reply:
fixed